### DISCUSSION TITLE
ckb的rpc官网中对submit_block例子params中"example"标签的疑惑

### DISCUSSION THREAD

**Discussion by ruyichang** [nervosnetwork/ckb#2411] [General]
在ckb的rpc官网中对submit_block例子params里面有个"example",的标签
---
```
 "method": "submit_block",
    "params": [
        "example",
        {
            "header": {
                "compact_target": "0x1e083126",
           ...
           }
       }
...
```
---

我在testnet中按照例子可以出块，请问在mainnet的submit_block的时候这个标签也是example吗？

这个example在这有点迷惑，容易和rpc的例子的含义混淆。

**Comment by ruyichang** [comment]
这个example的位置是work_id的string吗？

**Comment by doitian** [comment]
是 work_id，用 get_template 里返回的 work_id 原因填会去就可以了

**Comment by Lich-2016** [comment]
没有找到关于get_template  的文档，node的 help里面没有相关的说明

**Comment by doitian** [comment]
如果是要实现 miner，这里的 Python 示例可能能帮助到您，实现的是把 block template 转成可以提交的 block

https://github.com/doitian/ckb-sdk-python/blob/fb867293753ed8ed1b63ffeb296d61bd572df0ff/ckb/block.py#L45

**Comment by Lich-2016** [comment]
谢谢，非常清楚
